.prettierrc

```diff
@@ -1,6 +1,14 @@
 {
-    "printWidth": 80,
-    "proseWrap": "always",
-    "singleQuote": true,
-    "trailingComma": "all"
-  }
+  "printWidth": 80,
+  "proseWrap": "always",
+  "singleQuote": true,
+  "trailingComma": "all",
+  "overrides": [
+    {
+      "files": "CHANGELOG.md",
+      "options": {
+        "proseWrap": "preserve"
+      }
+    }
+  ]
+}
```

Comment:
just to confirm: Is this on purpose? No objections btw

Reply:
Yeah, I got annoyed by prettier fully re-formatting the changelog when formatting-on-save. This causes there to not be any changes anymore!